packages/admin/admin-generator/src/commands/generate/generate-command.ts

```diff
@@ -230,3 +234,10 @@ export async function runFutureGenerate(filePattern = "src/**/*.cometGen.ts") {
         }
     }
 }
+
+export const generateCommand = new Command("generate")
+    .option("-f, --file <file>", "path to config file or glob pattern to generate specific files")
+    .action(async ({ file: filePattern }: { file?: string }) => {
+        console.log("️️️⚠️️️⚠️️️⚠️️️  Admin Generator is still experimental and in beta phase.  ⚠️️️⚠️️️⚠️️️\n\n");
```

Comment:
Do we really want to log this every time?

Reply:
in my opinion it's good to mention that - so people know it's in beta and not fully stable.

Reply:
I'm fine with it. We should try to get out of beta soon anyway 😁